services/movie_session.py

```diff
@@ -41,4 +42,17 @@
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
```

Comment:
The function `delete_movie_session_by_id` correctly handles the case where the movie session does not exist by raising a `ValueError`. This is a good practice to inform the caller about the non-existence of the session.